api/src/main/java/org/pmiops/workbench/config/WebMvcConfig.java

```diff
@@ -48,18 +49,28 @@
   @Bean
   @RequestScope(proxyMode = ScopedProxyMode.DEFAULT)
   public UserAuthentication userAuthentication() {
-    return (UserAuthentication) SecurityContextHolder.getContext().getAuthentication();
+    Authentication auth = SecurityContextHolder.getContext().getAuthentication();
+    if (auth == null) {
+      return null;
+    }
+    return (UserAuthentication) auth;
   }
 
   @Bean
   @RequestScope(proxyMode = ScopedProxyMode.DEFAULT)
   public Userinfoplus userInfo(UserAuthentication userAuthentication) {
+    if (userAuthentication == null) {
+      return null;
+    }
     return userAuthentication.getPrincipal();
   }
 
   @Bean
   @RequestScope(proxyMode = ScopedProxyMode.DEFAULT)
   public DbUser user(UserAuthentication userAuthentication) {
+    if (userAuthentication == null) {
+      return null;
```

Comment:
Could we make a fake `DbUser` instance to represent the system?

Reply:
Also, if we go with this approach, it would help to mark both `userAuthentication` and the method itself with `@Nullable`.

Reply:
I might make more of a case for it if we were just beginning to define the notion of users and sessions, but in any case I think it's too late in the game to shoehorn one in. It would have the nice property of avoiding refactoring any injection sites, though.